contracts/crowdsale/price/CapStagedCrowdsale.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^ 0.4.24;
+
+import "../Crowdsale.sol";
+import "../../math/SafeMath.sol";
+import "../../ownership/Ownable.sol";
+
+
+/**
+  * @title CapStagedCrowdsale
+  * @dev Extension of Crowdsale contract that changes the price of tokens regarding of raised ETH. Each stage of crowdsale is defined in StageLimit (cap limit for the stage) and stageRate (stage rate)
+  * Stages must be inserted from first to last with increasing stageLimit.
+  */
+contract CapStagedCrowdsale is Crowdsale, Ownable {
+  using SafeMath for uint256;
+  using SafeMath for uint;
+  bool isActive = false;
+
+  struct Stage {
+    uint256 stageLimit;
```

Comment:
I think we can call these `limit` and `rate`, they are already part of a `Stage` (e.g. `stage.limit` as opposed to `stage.stageLimit`).

Reply:
I've changed this variables according to your suggestion.